boards/shields/seeed_xiao_round_display/seeed_xiao_round_display.overlay

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2024, Nicolas Goualard <nicolas.goualard@sfr.fr>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+    chosen {
+        zephyr,display = &gc9a01_xiao_round_display;
+    };
+
+    vbatt {
+        compatible = "voltage-divider";
+        io-channels = <&adc 0>;
```

Comment:
I don't think this works, as this would fail on e.g ESP32S3 XIAO where adc nodes are `adc0` and `adc1`, not `adc`?

@erwango what would be a correct way to do this?

Reply:
Idea would be to add a `xiao_adc` nodelabel (similarly to existing `xiao_serial` for instance) which would have to be defined by compatible boards.

Reply:
I've got a Xiao ESP32S3 laying around somewhere, I'll try to do some testing with it (hopefully this weekend)